src/trace/type.d.ts

```diff
@@ -28,8 +28,8 @@ export interface SpanFields {
   startTime: number;
   endTime: number;
   spanId: number;
-  spanLayer: number;
-  spanType: number;
+  spanLayer: string;
+  spanType: string;
```

Comment:
Are these working for the HTTP span receiver? FYI @arugal

Reply:
> Are these working for the HTTP span receiver? FYI @arugal

Yes, I  have checked the protocol https://github.com/apache/skywalking/blob/master/docs/en/protocols/Trace-Data-Protocol-v3.md. And I found the mistake from https://github.com/apache/skywalking-client-js/pull/76